chapters/101.rst

```diff
@@ -61,7 +61,7 @@ Executor
 
 
 Executors perform each task in an indexing or querying Flow. Jina's algorithmic logical units are put into different subclasses namely segmenter, ranker, encoder, crafter, classifier, indexer and evaluator. They all originate from the same parent - the Executor.
-To introduce a new algorithm to Jina, you can simply create a new Executor class by inheriting from the existing Executors, with focus on writing the algorithm itself.
+To introduce a new algorithm to Jina, you can simply create a new Executor class by inheriting from the existing Executors, with focus on writing the algorithm itself.  When you visualize a Flow in Jupyter notebook or Google Colab, each block in the graph is basically a part of the Executor (algorithm unit in Jina). This is the main interface that machine learning engineers and researchers would work with in Jina.  Jina has hundreds of classic and state-of-the-art Executors covering pre and post-processing indexing, ranking and encoding.
```

Comment:
Rewrite:

Executors are Jina's algorithmic logical units that perform each task in an indexing or querying Flow, and are the most common interface for machine learning engineers and researchers. 

Executors are organized into different subclasses, namely segmenter, ranker, encoder, crafter, classifier, indexer and evaluator. Jina comes with [hundreds of classic and state-of-the-art Executors](https://docs.jina.ai/chapters/all_exec/) out of the box.

To use a new algorithm with Jina, you can [create a new Executor class](https://docs.jina.ai/chapters/extend/executor/) by inheriting from existing Executors. This lets you focus on using the algorithm itself rather than worrying about implementation details.

If you use [Google Colab](https://jina.ai/2021/03/17/Jina-now-with-Google-Colab.html) or a Jupyter notebook to visualize a Flow, each block in the graph is a **part of the Executor**.



---

Re: the bold part above: Is each block:
- an Executor?
- part of an Executor?

As soon as that's cleared up, unbold the text and we're good :)

Reply:
Hang on. *Does* Jina come with **hundreds** of Executors out of the box? On [this page](https://docs.jina.ai/chapters/all_exec/) we only list 68

Reply:
yes, we have 100+ executors, the page in the docs is outdated - we can make update that too

Reply:
each part is an executor - since the concept of a pod basically is analogous to an executor